client/src/redux/reducers/allocations.reducers.js

```diff
@@ -68,18 +74,34 @@ export function allocations(state = initialState, action) {
           ...action.payload.loading
         }
       };
-    case 'SET_SEARCH_TERM':
+    case 'MANAGE_USERS_INIT':
+      return {
+        ...state,
+        loadingUsernames: {
```

Comment:
Could we rename `loadingUsernames` to be more descriptive? Something like `projectLoadingMap`